src/Symfony/Component/Security/Core/Encoder/BCryptPasswordEncoder.php

```diff
@@ -19,6 +19,8 @@
  */
 class BCryptPasswordEncoder extends BasePasswordEncoder
 {
+    const MAX_PASSWORD_LENGTH = 72;
```

Comment:
Maybe we could add a comment above this constant to show that this limit is not arbitrary. Something like this:

```php
// When using bcrypt algorithm, PHP's password_hash() truncates the password length to a maximum of 72 chars.
const MAX_PASSWORD_LENGTH = 72;
```